src/main/java/br/com/apppersonal/apppersonal/service/PersonalService.java

```diff
@@ -0,0 +1,134 @@
+package br.com.apppersonal.apppersonal.service;
+
+import br.com.apppersonal.apppersonal.exceptions.ParameterNullException;
+import br.com.apppersonal.apppersonal.exceptions.UserNotFoundException;
+import br.com.apppersonal.apppersonal.model.Dto.UserPersonalDto;
+import br.com.apppersonal.apppersonal.model.entitys.UserEntity;
+import br.com.apppersonal.apppersonal.model.repositorys.UserRepository;
+import br.com.apppersonal.apppersonal.security.Role;
+import br.com.apppersonal.apppersonal.utils.ApiResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class PersonalService {
+    private UserRepository userRepository;
+
+    @Autowired
+    public PersonalService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public ResponseEntity<?> associateUserWithPersonal(Long userId, Long personalId) {
```

Comment:
Adicionar comentário referente a função.

```suggestion
    /**
    * Método para asssociação de um personal trainer a um aluno
    *
    * @param  Long userId
    * @param  Long personalId
    * @return  ResponseEntity
    */
    public ResponseEntity<?> associateUserWithPersonal(Long userId, Long personalId) {
```